### PR TITLE
Second Milestone

### PR DESCRIPTION
**Pull request by orestis-kaplan**

### CODE REVIEW

File: src/containers/BooksForm.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const bookCategories = ["Action", "Biography", "History", "Horror", "Kids", "Learning", "Sci-Fi"];
+
+const BooksForm = () => (
+    <form>
+        <label for="title-input">Title</label>
+        <input id="title-input" type="text"/>
+        <select>
+            {bookCategories.map(category => (
+                <option value="category">{category}</option>
```
Comment (BalintMendli) [inline src/containers/BooksForm.js]: Lists in react need to have a unique `key` property in order to identify them for performance reasons. See the [docs](https://reactjs.org/docs/lists-and-keys.html#keys). 
So let's add a `key` prop to the mapped component.

File: src/containers/BooksList.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const BooksList = ({books}) => (
+    <div>
+        <table>
+            <tr>
+                <th>Book Id</th>
+                <th>Title</th>
+                <th>Category</th>
+            </tr>
+                {books.map(book => (                    
+                    <Book />
```
Comment (BalintMendli) [inline src/containers/BooksList.js]: Let's add a `key` property here as well.

**Review by BalintMendli — CHANGES_REQUESTED** [review]
Hi @userman95 ,

Great job in this milestone. 👏 

Please read my comments below and make the appropriate changes.

Also let's set up all import and exports properly and make sure the application works.

When done please submit a review request.

If you have any questions reach out to me on Slack.

**Review by SunnyAsar — APPROVED** [review]
# Hi @userman95 😃 
### update Looks good! please proceed 🚀 

## You May Pass 👍

**Pull request merged**